src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -320,6 +324,14 @@ class ClusterMonitorActor(val cluster: Cluster,
     } yield result
   }
 
+  private def isProxyAvailable(clusterTool: ClusterTool): Future[Boolean] = {
+    val toolDAO : ToolDAO = clusterTool match {
+      case ClusterTool.Jupyter => jupyterProxyDAO
+      case _ => rstudioProxyDAO // there are only two tools and it is impossible to create a cluster with no tools
```

Comment:
i would still recommend enumerating both of our supported tools _and_ throwing an exception in the case of an unrecognized tool

the benefit is that when we add a third tool, this bit of code will immediately fail and we can update it instead of it being masked because it's defaulting to RStudio. in other words, it means that the code will be more resilient to human-error

Reply:
good point!